### DISCUSSION TITLE
[Project Beta] Allow the filter input to start out hidden / collapsed when the view is put into focus

### DISCUSSION THREAD

**Discussion by clumsycomputer** [community/community#19789] [Projects and Issues]
#### How I setup views
- View names represent underlying filter configuration
- View filters are rarely updated after view setup

#### Improvement request
- Allow the filter input to start out hidden / collapsed when the view is put into focus

#### Requested improvement benefits
- Reduced clutter and noise resulting from filter configuration

**Comment by themiralay** [comment]
[This present issue](https://github.com/github-community/community/discussions/7575) is probably most popular issue with regard to tasks in Github Projects (Beta)!
 
 Other (possible) duplicates are:
 
 * [[Projects Beta] Show number of tasks vs completed task after issue title #16616](https://github.com/orgs/github-community/discussions/16616)
 * [[Projects Beta] Show x of n tasks on issue #13033](https://github.com/orgs/github-community/discussions/13033)
 * [[Projects beta] Task list progress #16721](https://github.com/orgs/github-community/discussions/16721)
 * [Show percent tasks complete in a new projects Beta field #12050](https://github.com/orgs/github-community/discussions/12050)

**Reply by clumsycomputer** [reply]
Apologies if I'm misinterpreting your response or if I failed to communicate effectively, but it seems we are talking about different things
